extensions/ql-vscode/package.json

```
@@ -293,6 +293,11 @@
           "scope": "window",
           "minimum": 0,
           "description": "Report a warning for any join order whose metric exceeds this value."
+        },
+        "codeQL.allowHttp": {
```

Please make it clear that this is related to database downloads only.

```suggestion
        "codeQL.databaseDownload.allowHttp": {
```